out/out_test.go

```diff
@@ -44,6 +44,7 @@ var _ = Describe("Out", func() {
 
 		BeforeEach(func() {
 			source = map[string]interface{}{}
+			source["interval"] = "1m"
```

Comment:
Is this a requirement for out i.e. if not setting `interval` or `start/stop` there wont be version returned? If thats the case maybe we should update the doc for `out` as well.

Reply:
I was actually thinking about the `out` function this morning.

In reality, you must either specify an `interval` or a `start`/`stop` range in the source configuration for `check` to work. I think we covered that fix in #50. Do you think we need an explicit callout for `out` on top of that?


Related to the `out` changes, I realized that there's a bug in the existing implementation - `out` may print a `0` value.

A couple of options:

1. Revert my changes, `out` returns the current timestamp - This would mean that `out` continues to generate versions that are not detectable by `check`. (😒)
1. Leave my changes as-is - `out` will always generate versions that are detectable by `check`, but may also generate a `0` version. (👎)
1.Enhance `TimeLord.Latest` - add another return value (i.e. `error`) such that the returned `time.Time` always represents the most recent value and the `error` indicates the non-fatal exceptional scenario when outside of the defined range with no previous, valid version. Update `check` to key off the presence of an `error` as needed. (👍 )

Clear as mud?

Reply:
To me 1/2 have obivious shortcomings so 3 looks like the only option. 

If an error is returned from TimeLoad.Latest, how would the `out` response look like in this case?

Reply:
I can't envision a scenario in which `out` would care about an error response. It would just care about the timestamp returned as part of the same function call.

The way I'm thinking about this, `check` would only care about the presence of an error and use it to omit the latest version (and return an empty list).

Reply:
So if neither interval nor start/stop given for `out`, it will return `0` value in current implementation. If going with option 3, will `out` still return `0` and omitting the error?

Reply:
> if neither interval nor start/stop given

Honestly, I'm not sure what this scenario represents (in a business context). But I recognize that the existing implementation supports it.

I just pushed a fresh commit that maintains the existing behavior - if neither an `interval` nor a `start`/`stop` range is provided in the configuration for `out`, it will return the current timestamp (and no error).